Spigot-API-Patches/0299-Add-corner-getter-to-WorldBorder.patch

```diff
@@ -0,0 +1,59 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TwoLeggedCat <80929284+TwoLeggedCat@users.noreply.github.com>
+Date: Wed, 12 May 2021 22:35:47 -0500
+Subject: [PATCH] Add corner getter to WorldBorder
+
+
+diff --git a/src/main/java/org/bukkit/WorldBorder.java b/src/main/java/org/bukkit/WorldBorder.java
+index afb7b136b461202026290624836446cff9f9e45d..b0c1585d8af6a3e2d2fe4c69c205ed82294d693a 100644
+--- a/src/main/java/org/bukkit/WorldBorder.java
++++ b/src/main/java/org/bukkit/WorldBorder.java
+@@ -54,6 +54,15 @@ public interface WorldBorder {
+      */
+     public void setCenter(@NotNull Location location);
+ 
++    //Paper start
++    /**
++     * Returns the specified corner of the WorldBorder
++     * @param corner Which corner to return
++     * @return Location of corner where y is always the minimum height of the world, or null if Corner is null
```

Comment:
```suggestion
+     * @return Location of corner where y is always the minimum height of the world
```
null is illegal, and it won't even return null, it'll just throw

Reply:
Ah yes, it used to do that and then I changed the method body without updating the Javadocs